### PR TITLE
feat dark theme

### PR DESCRIPTION
**Pull request by hsnice16**
This PR will add the code for the dark theme.

Major changes are done in the following files:
- app.css
- app.js

And all the HTML files has same changes, you can review code in any one of your want.

Here is this how it looks,
image

image

Please review this PR, and give your feedback.

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+});
+
+(() => {
+  const theme = localStorage.getItem("poshUITheme");
+
+  if (theme) {
+    htmlElement.setAttribute("data-theme", "dark");
+    moonIcon.classList.add("fas");
+  }
+})();
```
Comment (b0llu) [inline js/app.js:24]: easily understandable JS code. Great work

**Review by b0llu — APPROVED** [review]
Great work on adding Dark Theme to your CL!

**Review by keshavkumar-in — APPROVED** [review]
Just had a suggestion Himanshu, please check your contrast ratio for the texts and buttons.

**Comment by hsnice16** [comment]
Thank you for reviewing this

**Pull request merged**